src/App.tsx

```diff
@@ -1,52 +1,36 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list

Reply:
Done